docs/conf.py

```diff
@@ -28,6 +28,10 @@
 cwd = os.getcwd()
 project_root = os.path.dirname(cwd)
 
+# On Travis python3-gi is installed into folllowing directory, which
+# is not added to path because Travis uses virtualenv.
+sys.path.insert(0, '/usr/lib/python3/dist-packages')
```

Comment:
Virtualenv’s site-packages functionality is defined in [activate_this.py](https://github.com/pypa/virtualenv/blob/263283be88061b965ead9ebff47c37021d7d3f62/virtualenv_embedded/activate_this.py#L18)